applications/secux_hardware_wallet_support_polkadot_transactions_and_staking.md

```diff
@@ -0,0 +1,188 @@
+|     0e. | Article                                 | We will create and publish an in-depth article that explores the various supports and functionalities of the Polkadot network. Additionally, we will enhance the article's reach through a co-marketing initiative on social media with Polkadot. This strategic partnership will not only educate our audience about Polkadot's capabilities but also amplify our content's visibility and engagement across platforms.      |
+|      1. | Periodically Meeting                    | Collaborative Efforts and Communication, Polkadot Integration of SecuX SDK support.                                                                                                                                                                                                                                                                                                                                           |
+|      2. | SecuX SDK Polkadot support npm version  | Development of SDK API for Polkadot.js.                                                                                                                                                                                                                                                                                                                                                                                       |
+|      3. | SecuX SE Firmware version number        | Development of Secure Element (SE) Firmware.                                                                                                                                                                                                                                                                                                                                                                                  |
+|      4. | SecuX MCU Firmware version number       | Development of MCU Firmware including transection confirmation, transection details, account balance.                                                                                                                                                                                                                                                                                                                         |
+|      5. | SecuX SDK Polkadot integration tutorial | Provide SDK API for Polkadot.js.                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
The spec doesn't seem to match the name of the deliverable. That being said, I don't think a separate tutorial to integrate SecuX into Polkadot.js Apps is necessary. Documentation and testing guides should be included in 0b and 0c.

Reply:
ok, it is included in 0b and 0c, and it has been removed.